modules/httpcontrol/index.js

```diff
@@ -0,0 +1,123 @@
+import { EventEmitter } from 'events';
+import http from 'http';
+import querystring from 'querystring';
+
+/**
+ * Receives http requests from another peer.
+ */
+class HttpControl extends EventEmitter {
```

Comment:
I put logs for the development purpose, but I guess some of them can be removed now. What sort of lower-level logging would you suggest?